resources/testharness.js

```diff
@@ -793,6 +793,11 @@ policies and contribution forms [3].
     function done() {
         if (tests.tests.length === 0) {
             tests.set_file_is_test();
+            tests.status.status = tests.status.ERROR;
```

Comment:
@jugglinmike why aren't the `tests.set_file_is_test()` calls being removed, is this not the final PR in this process?

When done, I'd expect there to only be a single `tests.set_file_is_test()` call, and that's when the explicit opt-in is used.

The net effect would be that some tests with OK harness status and FAIL subtest would turn into ERROR harness status and no subtest.